src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterSearchParam;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/character")
```

Comment:
Plural form is preferred according to REST naming convention
```suggestion
@RequestMapping("/api/characters")
```